packages/smart-contracts/src/lib/artifacts/ChainlinkConversionPath/index.ts

```diff
@@ -45,6 +45,10 @@ export const chainlinkConversionPath = new ContractArtifact<ChainlinkConversionP
           address: '0xEEc4790306C43DC00cebbE4D0c36Fadf8634B533',
           creationBlockNumber: 10141032,
         },
+        arbitrumtestnet: {
```

Comment:
You need to use `''` in order to use `-` as object key name.
```
'arbitrum-testnet': {}
```

Reply:
Thanks for the tip, it's corrected